Feature/138

jaimemosber opened the pull request:
Tests de importación de censo a partir de archivo CSV.

jaimemosber committed:
TEST: Positive test for import census from CSV
Positive test for import census from CSV done

jaimemosber committed:
TEST: Positive and negative test for import census from CSV
Positive test for import census with groups from CSV done and negative cases tested too

jaimemosber committed:
FIX: Minor fixes in tests
Some minor fixes done in tests

jaimemosber committed:
FIX: Some minor conflicts fixed
Fixed some minor conflicts for merging

JaimeBorregoConde commented on decide/census/tests.py:
```diff
@@ -73,3 +88,155 @@ def test_destroy_voter(self):
+        self.base.setUp()
+
+        options = webdriver.ChromeOptions()
+        options.headless = False
```
Cambiar este parámetro a True

jaimemosber replied:
Perfecto. Ya está actualizado.

JaimeBorregoConde replied:
perfecto

jaimemosber committed:
FIX: webdriver.ChromeOptions().headless now is True
Minor fix. Now that parameter is true, so the navigator is not showing the tests.

JaimeBorregoConde reviewed (dismissed):
todo arreglado y funcionando

JaimeBorregoConde committed:
Fix: Problems with merge
Merging feature/138 with module/censo

JaimeBorregoConde reviewed (approved):
todo arreglado
